contracts/ProviderStrategy.sol

```diff
@@ -28,27 +34,25 @@ contract ProviderStrategy is BaseStrategy {
     bool public takeProfit;
     bool public investWant;
 
-    constructor(address _vault, address _joint) public BaseStrategy(_vault) {
-        _initializeStrat(_joint);
+    constructor(address _vault) public BaseStrategy(_vault) {
+        _initializeStrat();
     }
 
     function initialize(
         address _vault,
         address _strategist,
         address _rewards,
-        address _keeper,
-        address _joint
+        address _keeper
     ) external {
         _initialize(_vault, _strategist, _rewards, _keeper);
-        _initializeStrat(_joint);
+        _initializeStrat();
     }
 
-    function _initializeStrat(address _joint) internal {
+    function _initializeStrat() internal {
         require(
             address(joint) == address(0),
```

Comment:
if you are removing the joint from this method, it doesn't make sense to require this.